additionalResources.md

```diff
@@ -0,0 +1,96 @@
+# Additional Resources
+
+## Javascript
+Not being fluent with ES2015/2016, and not comfortable with pure array manipulation techniques (`.map`, `.filter`, `.reducer`) hurts a lot when using React, and you should first focus on that.
+- Wes Bos did a great series with [30 exercices in 30 days](https://javascript30.com/) to level up in JavaScript
+- [ES2016 in depth](https://ponyfoo.com/articles/tagged/es6-in-depth)
+
+## Functional Programming (FP)
+- [Composition over inheritance](https://www.youtube.com/watch?v=wfMtDGfHWpA): really cool video (you can check everything he did) on the differences between inheritance and composition
+- [Mostly Adequate Guide to Functional Programming](https://github.com/MostlyAdequate/mostly-adequate-guide). Free ebook on the concepts of FP (from broad concepts (function composition, pointfree programming, declarative) to category theory (maybes, monads, etc.))
+- Eric Eliott writes a lot of cool blog posts about JavaScript in general, but focuses mainly about the benefits of FP on [Medium](https://medium.com/@_ericelliott/latest), for instance:
+    - [10 interview questions every javascript developer should know](https://medium.com/javascript-scene/10-interview-questions-every-javascript-developer-should-know-6fa6bdf5ad95#.w7316qunx)
+    - [What is a closure?](https://medium.com/javascript-scene/master-the-javascript-interview-what-is-a-closure-b2f0d2152b36#.rg6t8we4i)
+    - [What is a function composition?](https://medium.com/javascript-scene/master-the-javascript-interview-what-is-a-closure-b2f0d2152b36#.rg6t8we4i)
+    - [What is a pure function?](https://medium.com/javascript-scene/master-the-javascript-interview-what-is-a-closure-b2f0d2152b36#.rg6t8we4i)
+    - [Difference between class and prototype](https://medium.com/javascript-scene/master-the-javascript-interview-what-is-a-closure-b2f0d2152b36#.rg6t8we4i)
```

Comment:
Tous les liens sont identiques entre what is a closure & difference between class and prototype